tests/conftest.py

```diff
@@ -0,0 +1,246 @@
+"""Module containing helper functions for testing PyTeal Utils."""
+
+from base64 import b64decode
+from typing import List, Optional
+
+from algosdk import account, encoding, kmd, mnemonic
+from algosdk.future import transaction
+from algosdk.v2client import algod, indexer
+from pyteal import Cond, Expr, Int, Mode, Seq, Txn, compileTeal
+
+## Clients
+
+
+def _algod_client(algod_address="http://localhost:4001", algod_token="a" * 64):
+    """Instantiate and return Algod client object."""
+    return algod.AlgodClient(algod_token, algod_address)
+
+
+def _indexer_client(indexer_address="http://localhost:8980", indexer_token="a" * 64):
+    """Instantiate and return Indexer client object."""
+    return indexer.IndexerClient(indexer_token, indexer_address)
+
+
+def _kmd_client(kmd_address="http://localhost:4002", kmd_token="a" * 64):
+    """Instantiate and return a KMD client object."""
+    return kmd.KMDClient(kmd_token, kmd_address)
+
+
+# Env helpers
+
+
+class Account:
+    address: str
+    private_key: Optional[str]  # Must be explicitly set to None when setting `lsig`.
+    lsig: Optional[transaction.LogicSig] = None
+
+    def __init__(
+        self,
+        address: str,
+        private_key: Optional[str],
+        lsig: Optional[transaction.LogicSig] = None,
+    ):
+        self.address = address
+        self.private_key = private_key
+        self.lsig = lsig
```

Comment:
```suggestion
        self.address: str = address
        self.private_key: Optional[str] = private_key
        self.lsig:  Optional[transaction.LogicSig] = lsig
```

Don't really need this because it's really clear from the param's, but it's possible to hint again.